### PR TITLE
Add the isNonNull function.

### PR DESCRIPTION
**Pull request by vision-sbm**
http://graphite.readthedocs.org/en/latest/functions.html#graphite.render.functions.isNonNull

### CODE REVIEW

File: expr.go [hunk 1]
```diff
@@ -935,6 +935,29 @@ func evalExpr(e *expr, from, until int32, values map[metricRequest][]*metricData
 
 		return results
 
+	case "isNonNull", "isNotNull": // isNonNull(seriesList), isNotNull(seriesList)
+
+		_, err := getSeriesArg(e.args[0], from, until, values)
```
Comment (dgryski) [inline expr.go]: Do you even need to call `getSeriesArg()` here?

File: expr.go [hunk 2]
```diff
@@ -935,6 +935,29 @@ func evalExpr(e *expr, from, until int32, values map[metricRequest][]*metricData
...
+
+		e.target = "isNonNull"
+
+		if err != nil {
+			return nil
+		}
+
+		return forEachSeriesDo(e, from, until, values, func(a *metricData, r *metricData) *metricData {
+			for i, _ := range a.Values {
```
Comment (dgryski) [inline expr.go]: Don't need the _ here. `for i := range a.Values` will do what you want.

**Comment by vision-sbm** [comment]
Did a rewrite of the lines in question. I was initially worried about not checking for an err, but I see that other existing functions don't take it into consideration either.

**Comment by dgryski** [comment]
LGTM

**Pull request merged**